product_top_category/models/product_category.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import api, fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    def _get_top_category(self):
+        self.ensure_one()
+        if self.parent_id:
+            return(self.parent_id._get_top_category())
```

Comment:
Why the parentheses ?

Reply:
ah, don't know, it just came naturally ^^ I remove them